crypto/cmac/cmac.c

```diff
@@ -161,7 +162,8 @@ int CMAC_Init(CMAC_CTX *ctx, const void *key, size_t keylen,
 int CMAC_Update(CMAC_CTX *ctx, const void *in, size_t dlen)
 {
     const unsigned char *data = in;
-    int bl;
+    int bl, max_burst_block, cipher_block;
```

Comment:
Nit: max_burst_block and cipher_block should be `size_t` and they should be called `max_burst_blocks` and `cipher_blocks` to make clear they indicate numbers of blocks.

Reply:
Thanks for advice. I've updated the patch.